tests/test_cli_run.py

```diff
@@ -0,0 +1,140 @@
+"""This does not cover all CLI parameters defined in caper/caper_args.py.
+Google Cloud Platform is tested in test_cli_server_client.py.
+However, other cloud (aws) and HPCs (slurm/sge/pbs) are not tested.
+
+In this testing module, 'caper run' is tested with a local backend.
+
+See test_cli_server_client.py for 'caper server/submit/...'.
+We will use gcp (Google Cloud Platform) backend to test server-client
+functions.
+"""
+import json
+import os
+
+import pytest
+
+from caper.cli import main as cli_main
+from caper.wdl_parser import WDLParser
+
+from .example_wdl import make_directory_with_wdls
+
+
+def test_wrong_subcmd():
+    cmd = ['wrong_subcmd']
+    with pytest.raises(SystemExit):
+        cli_main(cmd)
+
+
+def test_mutually_exclusive_params(tmp_path):
+    make_directory_with_wdls(str(tmp_path))
+
+    # mutually exclusive params
+    cmd = ['run', str(tmp_path / 'main.wdl'), '--docker', '--singularity']
+    with pytest.raises(ValueError):
+        cli_main(cmd)
+    cmd = ['run', str(tmp_path / 'main.wdl'), '--docker', '--soft-glob-output']
+    with pytest.raises(ValueError):
+        cli_main(cmd)
+
+
+def test_run(tmp_path, cromwell, womtool, debug_caper):
```

Comment:
It would be good to have a `pytest` mark here to indicate that this is an integration test/slow to run:
```suggestion
@pytest.mark.integration
def test_run(tmp_path, cromwell, womtool, debug_caper):
```

Reply:
Added `pytest.mark` as you suggested.